USFMToolsSharp.Renderers.Docx/Extensions/XWPFRunExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NPOI.XWPF.UserModel;
+
+
+namespace USFMToolsSharp.Renderers.Docx.Extensions
+{
+    public static class XWPFRunExtensions
+    {
+        public static void StyleRun(this XWPFRun run, StyleConfig styles)
```

Comment:
You should actually create an extension method for CreateRun on paragraph that does this instead of adding another function call.